__tests__/app.test.js

```diff
@@ -27,6 +27,66 @@ describe("/api/categories", () => {
   });
 });
 
+describe("/api/reviews/:reviews_id", () => {
+  test("Get request to reivew_id responds with correctly formatted review object", () => {
+    return request(app)
+      .get("/api/reviews/6")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(
+          expect.objectContaining({
+            review_id: expect.any(Number),
+            title: expect.any(String),
+            review_body: expect.any(String),
+            designer: expect.any(String),
+            review_img_url: expect.any(String),
+            votes: expect.any(Number),
+            category: expect.any(String),
+            owner: expect.any(String),
+            created_at: expect.any(String),
+          })
+        );
+      });
+  });
```

Comment:
Don't forget to check the length of the array matches the length of the test data array. If it's an empty array it will pass automatically.